src/Storage/Configuration/PlatformSettings.cs

```diff
@@ -0,0 +1,31 @@
+using System.Runtime.Serialization;
+
+namespace Altinn.Platform.Storage.Configuration
+{
+    /// <summary>
+    /// Represents a set of configuration options when communicating with the platform API.
+    /// Instances of this class is initialised with values from app settings. Some values can be overridden by environment variables.
+    /// </summary>
+    public class PlatformSettings
```

Comment:
@acn-sbuad @codemascot I suspect that the problem lies here. I'm not sure how the platformsettings was handled before. Now i see that this file does not have ApiAuthorizationEndpoint and therefore platformsettings does not read the config value ApiAuthorizationEndpoint. So when PEP tries to fetch the ApiAuthorizationEndpoint, its null. Therefore the uri format exception.

Reply:
Thanks for pointing that out! 👍 Here is the fix commit: https://github.com/Altinn/altinn-storage/pull/376/commits/25aba89a59880ff8632d11269a6d696d6cb0be04 🙂